geometry_msgs/PoseWithCovarianceStamped[] poses
std_msgs/Bool only_for_tracking
